src/modules/emote_menu/stores/emote-menu-store.js

```diff
@@ -33,13 +34,45 @@ function computeScore({totalUses, recentUses}) {
   return Math.floor((totalUses * frecency) / recentUses.length);
 }
 
+const TWITCH_EMOTE_USAGE_HISTORY_KEY = 'twilight.emote_picker_history';
+
+function loadDefaultFrequentEmotes() {
+  if (getPlatform() !== PlatformTypes.TWITCH || !storage.localStorageSupport) {
+    return DEFAULT_FREQUENT_EMOTES;
+  }
+
+  const history = storage.get(TWITCH_EMOTE_USAGE_HISTORY_KEY, null);
+  if (history == null) {
+    return DEFAULT_FREQUENT_EMOTES;
+  }
+
+  try {
+    const historyEntries = Object.entries(history);
+    if (historyEntries.length === 0) {
+      return DEFAULT_FREQUENT_EMOTES;
+    }
+    const emoteHistory = {};
+    for (const [emoteId, {uses, lastUpdatedAt}] of historyEntries) {
+      const serializedEmoteId = `${EmoteProviders.TWITCH}-${emoteId}`;
```

Comment:
nit: it would be great if this were shared with how we already generate other emote canonical ids. if we change how canonical ids are generated, this will break.